crates/driver/src/commit_reveal.rs

```diff
@@ -21,8 +27,9 @@ pub struct SettlementSummary {
     pub settled_orders: Vec<OrderUid>,
 }
 
-#[cfg_attr(test, mockall::automock)]
+// #[cfg_attr(test, mockall::automock)]
 #[async_trait::async_trait]
+#[cfg_attr(feature = "test", mockall::automock)]
```

Comment:
nit: Is this potentially some left-over testing code?

Reply:
Good catch. This came from experimenting with the `test` feature idea described in #410.